src/addresstype.cpp

```diff
@@ -94,6 +94,7 @@ bool ExtractDestination(const CScript& scriptPubKey, CTxDestination& addressRet)
     case TxoutType::MULTISIG:
     case TxoutType::NULL_DATA:
     case TxoutType::NONSTANDARD:
+        addressRet = CNoDestination(scriptPubKey);
```

Comment:
Comment for `ExtractDestination` function needs to be updated.

Reply:
Updated the comments